app/modules/node.c

```diff
@@ -448,6 +449,39 @@ static int node_compile( lua_State* L )
   return 0;
 }
 
+// Task callback handler for node.tasknow()
+static task_handle_t do_node_task_handle;
+static void do_node_task (task_param_t task_fn_ref, uint8_t prio)
+{
+  UNUSED(prio);
+  lua_State* L = lua_getstate();
+  lua_rawgeti(L, LUA_REGISTRYINDEX, (int)task_fn_ref);
+	luaL_unref(L, LUA_REGISTRYINDEX, (int)task_fn_ref);
```

Comment:
Something's odd about the indentation here & below. I'm not too fussed whether it's tabs or spaces, but consistency would be handy :)